web/src/features/fba/components/viz/FuelSummary.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react'
+import { FireShape, FireZoneThresholdFuelTypeArea } from 'api/fbaAPI'
+import { Box, Tooltip, Typography } from '@mui/material'
+import { groupBy, isUndefined } from 'lodash'
+import { DateTime } from 'luxon'
+import FuelDistribution from 'features/fba/components/viz/FuelDistribution'
+import { DataGridPro, GridColDef, GridRenderCellParams } from '@mui/x-data-grid-pro'
+import { useTheme } from '@mui/material/styles'
+
+export interface FuelTypeInfoSummary {
+  area: number
+  criticalHoursStart?: DateTime
+  criticalHoursStop?: DateTime
+  id: number
+  code: string
+  description: string
+  percent?: number
+  selected: boolean
+}
+
+interface FuelSummaryProps {
+  fuelTypeInfo: Record<number, FireZoneThresholdFuelTypeArea[]>
+  selectedFireZoneUnit: FireShape | undefined
+}
+
+const columns: GridColDef[] = [
+  {
+    field: 'code',
+    headerName: 'Fuel',
+    sortable: false,
+    width: 75,
+    renderCell: (params: GridRenderCellParams) => (
+      <Tooltip placement="right" title={params.row['description']}>
+        <Typography sx={{ fontSize: '0.75rem' }}>{params.row[params.field]}</Typography>
+      </Tooltip>
+    )
+  },
+  {
+    field: 'area',
+    flex: 3,
+    headerName: 'Area Distribution',
+    minWidth: 200,
+    sortable: false,
+    renderCell: (params: GridRenderCellParams) => {
+      console.log(params)
+      return <FuelDistribution code={params.row['code']} percent={params.row['percent']} />
+    }
+  }
+  // { field: 'hours', flex: 2, headerName: 'Critical Hours', minWidth: 150, sortable: false }
```

Comment:
```suggestion

```

Reply:
Fixed in https://github.com/bcgov/wps/pull/3792/commits/2025fad89a3a04d99b7c15d2956bb9b31b947584